src/lib/y2storage/proposal/autoinst_nfs_planner.rb

```diff
@@ -0,0 +1,235 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/proposal/autoinst_drive_planner"
+
+module Y2Storage
+  module Proposal
+    # This class converts an AutoYaST specification into a Planned::Nfs in order
+    # to set up a NFS filesystem.
+    class AutoinstNfsPlanner < AutoinstDrivePlanner
+      # Returns an array of planned NFS filesystems according to an AutoYaST specification
+      #
+      # @param drive [AutoinstProfile::DriveSection] drive section describing NFS filesystems
+      # @return [Array<Planned::Nfs>] Planned NFS filesystems
+      def planned_devices(drive)
+        # TODO: planned device with new format
+
+        planned_devices_old_format(drive)
+      end
+
+    private
+
+      # This hash defines mandatory profile values for a NFS share using the old AutoYaST style.
+      #
+      # Keys are the name of the section, and values are the mandatory values for such section.
+      #
+      # With old format, the partition section must contain a device and a mount value, e.g.:
+      #
+      # <drive>
+      #   <device>/dev/nfs</device>
+      #   <partitions>
+      #     <partition>
+      #       <device>192.168.56.1:/root_fs</device>
+      #       <mount>/</mount>
+      #     </partition>
+      #   </partitions>
+      # </drive>
+      OLD_FORMAT_MANDATORY_VALUES = { drive: [], partition: [:device, :mount] }.freeze
+
+      # Similar to {OLD_FORMAT_MANDATORY_VALUES}, but for the new AutoYaST style.
+      #
+      # TODO
```

Comment:
I would add a note pointing out that it is available in SLE 15 SP1. Otherwise, I would think it is just missing, and we need to implement it.

Reply:
OK, after an offline discussion, it is clear that we will implement the "export" feature later.